src/uct/tcp/sockcm/sockcm_ep.c

```diff
@@ -44,9 +47,173 @@ void uct_sockcm_ep_put_sock_id(uct_sockcm_ctx_t *sock_id_ctx)
     ucs_free(sock_id_ctx);
 }
 
+ucs_status_t uct_sockcm_ep_send_client_info(uct_sockcm_iface_t *iface, uct_sockcm_ep_t *ep)
+{
+    ssize_t transfer_len = 0;
+    ssize_t sent_len     = 0;
+    ssize_t sent_off     = 0;
+    uct_md_h dummy_md    = 0;
+    int retries          = 0;
+    ucs_status_t status;
+    unsigned num_resources;
+    uct_tl_device_resource_t *devices;
+    uct_sockcm_conn_param_t conn_param;
+
+    memset(&conn_param, 0, sizeof(uct_sockcm_conn_param_t));
+
+    status = uct_tcp_query_devices(dummy_md, &devices, &num_resources);
+    if (UCS_OK != status || num_resources == 0) {
```

Comment:
code style: plz use `()` for each statement, like: `(UCS_OK != status) || (num_resources == 0)`